source/includes/steps-starting-compass-individual-fields.yaml

```
@@ -156,6 +161,43 @@ content: |
                  canonicalization in Kerberos, see `this RFC document
                  <https://tools.ietf.org/html/rfc6806.html>`__.
 
+                 Authenticate as a Different Kerberos User on Windows
+                   When you authenticate with Kerberos on Windows, the
+                   :guilabel:`Principal` must be the same as the
+                   principal of the security context that
+                   |compass-short| is running. Normally, this is the
+                   logged-in user who is running |compass-short|.
+
+                   To authenticate as a different Kerberos user, you can
+                   run |compass| as the desired user, and instead
+                   specify the :guilabel:`Principal` for that user.
+
+                   .. example::
+
+                      To run |compass| as a user named ``admin``:
+
+                      .. code-block:: sh
```

The highlighting for this renders a little weirdly for me (specifically around that backtick, which it seems to interpret as an escape character. Perhaps you could change this to:
```suggestion
                      .. code-block:: text
```
Since the highlighting is neither useful nor important here?

---

Agreed. Updating.